include/Track.h

```diff
@@ -250,12 +250,16 @@ class TrackContentObjectView : public selectableObject, public ModelView
 	bool needsUpdate();
 	void setNeedsUpdate( bool b );
 
-	// Methods to remove, copy, cut, paste and mute selections of TCO views
-	void removeSelection( QVector<selectableObject *> so );
-	void copySelection( QVector<selectableObject *> so );
-	void cutSelection( QVector<selectableObject *> so );
-	void pasteSelection(); // TODO: Rethink naming?
-	void toggleMuteSelection( QVector<selectableObject *> so );
+	// Methods to remove, copy, cut, paste and mute a QVector of TCO views
```

Comment:
I think you should drop the `Active` from these names. Technically you can pass whatever QVector of TCOs you want and the function will copy/cut/remove/mute them, so I think it's more accurate that way.

Reply:
I also removed the `Active` from the method names, so they are now overloads of the individual TCOs methods. At first I thought they should be differentiated by name to make it less confusing to have the same name as other methods that works differently, but I agree the `QVector` argument should be enough for someone reading the code to understand what they are there for!